### PR TITLE
feat: update to new SDK 3.13

### PR DESCRIPTION
**Pull request by shumkov**
## Issue being fixed or feature implemented

In order to test against DP 0.13 networks we need to upgrade SDK to 3.13

## What was done?

* Increased mocha timeout to 250000 ms
* Reduced number of blocks to wait to 1 during wallet funding
* Initiate wallet in specific tests
* Set funding amount to 10
* Fixed Travis build

## How Has This Been Tested?

Ran the tests suite against Palinka.

## Breaking Changes

None.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: test/e2e/contacts.spec.js [hunk 1]
```diff
@@ -16,20 +15,19 @@ describe('e2e', () => {
...
+    let aliceClient;
 
     let bobIdentity;
     let bobContactRequest;
     let aliceIdentity;
     let aliceProfile;
     let aliceContactAcceptance;
 
-    let dataContractDocumentSchemas;
+    let dataContractDocumentDefintions;
```
Comment (thephez) [inline test/e2e/contacts.spec.js:30]: These document definitions are not the definitions set by `contract.setDefinitions()`, correct? What is the right way to refer to these 2 different things - "document definitions" and "definitions"? Seems like confusing terminology.

Comment (shumkov) [reply test/e2e/contacts.spec.js:30]: Yeah, agree. Document schemas?

Comment (thephez) [reply test/e2e/contacts.spec.js:30]: Maybe. That's what it was before this change :thinking:

File: package.json [hunk 2]
```diff
@@ -11,25 +11,24 @@
...
   "devDependencies": {
-    "@dashevo/dapi-client": "~0.12.0",
-    "@dashevo/dashcore-lib": "~0.18.0",
-    "@dashevo/dpp": "~0.12.1",
+    "@dashevo/dashcore-lib": "~0.18.4",
+    "@dashevo/dpp": "~0.13.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
-    "dash": "github:dashevo/dashjs#refactor-client-options",
+    "dash": "^3.13.0",
```
Comment (shumkov) [inline package.json:26]: ~

**Pull request merged**